src/browser/components/LoginModal.jsx

```diff
@@ -54,7 +53,10 @@ export default class LoginModal extends React.Component {
                 <FormControl
                   type='text'
                   placeholder='User Name'
-                  ref={this.usernameRef}
+                  value={this.state.username}
+                  onChange={(e) => {
```

Comment:
lets make this a function in the class something like
```js
setUser = (e) => { this.username = e.target.value }
```

then the `onChange` prop is `onChange=this.setUser`

Reply:
Done